composer.json

```diff
@@ -24,11 +24,10 @@
  		"psr-4": { "Michelf\\": "Michelf/" }
 	},
 	"require-dev": {
-		"phpspec/prophecy": "^1.6",
```

Comment:
There's no need for this dependency. See:
https://github.com/sebastianbergmann/phpunit/issues/5033#issuecomment-1221992857